requirements.json

```diff
@@ -282,6 +282,32 @@
       }
     }
   },
+  "rhamt": {
+    "name": "Red Hat Application Migration Toolkit",
+    "description": "",
```

Comment:
Description should be: "An assembly of open source tools that enables large-scale application migrations and modernizations"